### PR TITLE
ToPossessive 

### PR DESCRIPTION
**Pull request by wgimson**
To Possessive functionality added along with tests in new branch ToPossessive, covering cases of default, words ending in 's', common words ending in silent 's' and common possessive pronouns which require no changes. These latter two lists of words will almost certainly be appended to as time goes by.

Just looking at this code I can see that you guys are awesome developers and way beyond my level. Whether you accept this fix or not I welcome any criticism concerning the code and future advice for submitting pull requests and designing bug fixes in general. Thanks for looking!

### CODE REVIEW

File: src/Humanizer/TransformToPossessive.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        {
+            var len = input.Length;
+            var lastChar = input[len - 1];
+            return (char.ToUpper(lastChar) == 'S');
+        }
+
+        private static bool EndsInSilentS(string input)
+        {
+            var uppercaseInput = input.ToUpper().Trim();
+            return (_nounsEndingInSilentS.Contains(uppercaseInput));
```
Comment (hazzik) [inline src/Humanizer/TransformToPossessive.cs]: This will fail in turkish.

File: src/Humanizer/Humanizer.csproj [hunk 2]
```diff
@@ -244,9 +245,6 @@
   <ItemGroup>
     <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetPortable)' == 'false' ">
```
Comment (hazzik) [inline src/Humanizer/Humanizer.csproj]: Why this has been removed?

Comment (wgimson) [inline src/Humanizer/Humanizer.csproj]: Hm good question, it's present on my local system but not in my branched repository. And I don't have any changes to commit.

File: src/Humanizer/TransformToPossessive.cs (hunk 1 above)
Comment (wgimson) [inline src/Humanizer/TransformToPossessive.cs]: Hm reading up on localization now, not really experienced with that.

**Comment by onovotny** [comment]
What is the current status on this? @hazzik? Is this something that's complete and ready? Broken on some languages (Turkish)? Something that's beyond the scope of Humanizer due to it being grammar related?

**Comment by hazzik** [comment]
I think if we accept this, this needs to be moved to a language specific namespace. Something like Humanizer.English. Also, Inflector worth moving to a language specific namespace.

**Comment by onovotny** [comment]
@hazzik Do you think this should go into 2.0 or until the release after that?

**Comment by hazzik** [comment]
@onovotny I think after 2.0

**Comment by MehdiK** [comment]
This could be a great new addition but this PR has been sitting here for over two years @wgimson; so I'm going to close it. It would be great if you could please resolve the conflicts and resubmit the PR. Please also add the new method to readme documentation. cc @onovotny @hazzik

**Pull request closed without merging**